.circleci/config.yml

```diff
@@ -1340,6 +1340,27 @@ jobs:
           name: Verify Values Match
           command: |
             ./ops/scripts/ci-match-values-between-files.sh "<< parameters.file1_path >>" "<< parameters.pattern_file1 >>" "<< parameters.file2_path >>" "<< parameters.pattern_file2 >>"
+
+  kontrol-tests:
+    docker:
+      - image: runtimeverificationinc/kontrol:ubuntu-jammy-0.1.74
+    resource_class: xlarge
+    steps:
+      - checkout
+      - check-changed:
+          patterns: contracts-bedrock/test/kontrol,contracts-bedrock/src/L1/OptimismPortal\.sol
+      - run:
+          name: Run Kontrol Tests
+          command: |
+            ./packages/contracts-bedrock/scripts/kontrol.sh           
```

Comment:
Does this need to run in the context of the root of the repo? If no, we should run it in the context of the contracts package and wrap the call to the script in a pnpm command to follow suit of the other tests

Reply:
See
https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/scripts/generate-l2-genesis.sh#L10
On how to do this

Reply:
Sure thing the script was to address the ask for a script to wrap the steps taken to run the tool in the proper context. I can take a look at using pnpm instead.